lambda-samples/samples/docdb-scaleOnSchedule.py

```diff
@@ -0,0 +1,92 @@
+import json
+import os
+import boto3
+import random
+import logging
+import string
```

Comment:
Do you need string? I don't see it being used.

Reply:
It is used in line 72 : randString = "".join(random.choices(string.ascii_lowercase, k=6))